src/client/types.ts

```diff
@@ -18,6 +19,7 @@ export type EventFilter = {
   operationId?: string
   isFinal?: boolean
   isError?: boolean
+  status?: ScExecutionEventStatus
```

Comment:
this new "status" property is misleading, because it will only work for grpc..
The best solution would be to set it depending on "isFinal" value in the `getEvents`of grpcPulblicProvider.
about `ScExecutionEventStatus`i dont really get why we have a "Readonly" I don't think its relevant in getEvents endpoint.
so to resume:
```
let status = ScExecutionEventStatus.SC_EXECUTION_EVENT_STATUS_CANDIDATE
if(isFinal)
  status = ScExecutionEventStatus.SC_EXECUTION_EVENT_STATUS_FINAL

Reply:
done